src/main/java/org/webjars/taglib/LocateTag.java

```diff
@@ -126,8 +135,18 @@ public int doStartTag() throws JspException {
 		}
 
 		if (path != null) {
+			if (versionAgnostic) {
```

Comment:
Sorry I use to code with blank spaces instead of tab characters (and mixed them trying to minify changes). Fixed now.